From 5fb1e5a7e821c263570eba0cbd3784b17789db47 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jir=CC=8Ci=CC=81=20Techet?= <techet@gmail.com>
Date: Mon, 30 Dec 2024 19:21:19 +0100
Subject: [PATCH] Disable meson and forth regex parsers

These are extremely slow on macOS and make any editing pretty much unusable
---
 data/filedefs/filetypes.Meson.conf | 2 +-
 data/filedefs/filetypes.forth      | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/data/filedefs/filetypes.Meson.conf b/data/filedefs/filetypes.Meson.conf
index c97f8d153..0752182d0 100644
--- a/data/filedefs/filetypes.Meson.conf
+++ b/data/filedefs/filetypes.Meson.conf
@@ -8,7 +8,7 @@ identifiers=add_global_arguments add_global_link_arguments add_languages add_pro
 
 [settings]
 lexer_filetype=Python
-tag_parser=Meson
+tag_parser=
 extension=build
 comment_single=#
 mime_type=text/x-meson
diff --git a/data/filedefs/filetypes.forth b/data/filedefs/filetypes.forth
index 20b556ab6..17475ff7d 100644
--- a/data/filedefs/filetypes.forth
+++ b/data/filedefs/filetypes.forth
@@ -26,6 +26,7 @@ preword2=! c! @ c@ 2! 2@ and or xor invert negate / /mod mod rshift lshift
 [settings]
 # default extension used when saving files
 extension=fs
+tag_parser=
 
 # these characters define word boundaries when making selections and searching
 # using word matching options
-- 
2.45.2

